apis/fluentbit/v1alpha2/clusteroutput_types.go

```diff
@@ -100,7 +100,8 @@ type OutputSpec struct {
 	S3 *output.S3 `json:"s3,omitempty"`
 	// Gelf defines GELF Output configuration.
 	Gelf *output.Gelf `json:"gelf,omitempty"`
-
+	// OracleLogAnalytics defines OracleLogAnalytics Output configuration
+	OracleLogAnalytics *output.OracleLogAnalytics `json:"oracleLogAnalytics,omitempty"`
 	// CustomPlugin defines Custom Output configuration.
```

Comment:
Can you provide a link to the official plugin documentation? Is this the plugin?
https://docs.fluentbit.io/manual/pipeline/outputs/azure
![1696599488220](https://github.com/fluent/fluent-operator/assets/39892300/06364caf-4e08-4541-aaf5-27dbd53e44f7)

Reply:
> Can you provide a link to the official plugin documentation? Is this the plugin?

@adiforluls I can not find this OracleLogAnalytics output plugin in [outputs](https://docs.fluentbit.io/manual/pipeline/outputs), is this an official plugin?

Reply:
Yeah it's a new plugin that I wrote and is part if v2.1.10, but looks like I made a small mistake in my fluentbit docs contribution due to which the docs aren't visible. I'm fixing it and will post here after the docs are visible.